Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.18 AS builder
+FROM golang:1.20 AS builder
```

```suggestion
FROM golang:18 AS builder
```
looks like the integration tests fails because it depends on this version https://github.com/argoproj-labs/argocd-image-updater/actions/runs/4566893719/jobs/8060005060?pr=548